packages/regl-worldview/src/WorldviewContext.js

```diff
@@ -240,63 +244,104 @@ export class WorldviewContext {
       regl({ framebuffer: _fbo })(() => {
         // clear the framebuffer
         regl.clear({ color: [0, 0, 0, 1], depth: 1 });
+        let currentObjectId = 0;
+        const seenObjects = [];
+        const seenIds: Array<HitmapId> = [];
```

Comment:
should we call this type ObjectId or HitmapObjectId so it matches the "objects" terminology?

Reply:
I think `HitmapObjectId` makes sense because it is an object in the hitmap